code/modules/mob/living/carbon/diona_base.dm

```diff
@@ -0,0 +1,548 @@
+//This function is for code that is shared by diona nymphs and gestalt
+/mob/living/carbon/var/stored_energy
```

Comment:
They are in fact, entirely unnecessary and unused variables because i incorporated them into the dionastats datum. good catch